### ISSUE TITLE
Timeout in loadHooks needs to be configurable.

### ISSUE THREAD

**Issue by stefan1968** [wistityhq/strapi#63]
``` javascript
function loadHook(id, cb) {
  const timeoutInterval = 5000;
  let hookTimeout;
  ...
}
```

in File : https://github.com/wistityhq/strapi/blob/master/lib/private/loadHooks.js#L85

We need a way of setting the timeout via the strapi environment. This value is too small for remote connections, for example Amazon RDS.

**Comment by loicsaintroch** [comment]
I'll take a look at it this morning @stefan1968.

**Comment by loicsaintroch** [comment]
Just released the v1.5.4 without a hook timeout. We decided to remove it since it depends on the environment and we can't control it. This is the best option we have.
